File: android/app/src/main/java/com/smartexpense/android/data/remote/dto/response/ExpenseResponseDto.java
package com.smartexpense.android.data.remote.dto.response;

public class ExpenseResponseDto {
    private String id;
    private double amount;
    private String category;
    private String photoUrl;
    private String caption;
    private String expenseDate;

    public String getId() { return id; }
    public double getAmount() { return amount; }
    public String getCategory() { return category; }
    public String getPhotoUrl() { return photoUrl; }
    public String getCaption() { return caption; }
    public String getExpenseDate() { return expenseDate; }
}

File: android/app/src/main/kotlin/com/smartexpense/android/data/remote/dto/response/ExpenseResponseDto.kt
package com.smartexpense.android.data.remote.dto.response

import com.google.gson.annotations.SerializedName

data class ExpenseResponseDto(
    @SerializedName("id") val id: String,
    @SerializedName("amount") val amount: Double,
    @SerializedName("category") val category: String,
    @SerializedName("photoUrl") val photoUrl: String,
    @SerializedName("caption") val caption: String,
    @SerializedName("expenseDate") val expenseDate: String
)
